crates/sqllib/Cargo.toml

```diff
@@ -40,6 +40,7 @@ base64 = { workspace = true }
 uuid = { workspace = true, features = ["v4", "std"] }
 arcstr = { workspace = true, features = ["serde"] }
 ryu = { workspace = true }
+md-5 = { version = "0.10.6" }
```

Comment:
I suggest everything should be in the workspace, even if only used by one crate.

Reply:
I will move it